test/extended/openstack/loadbalancers.go

```diff
@@ -471,8 +471,15 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack][lb][Serial] The O
 			g.By("Test that the canary service is accessible through new ingressController")
 			route, err := oc.AdminRouteClient().RouteV1().Routes("openshift-ingress-canary").Get(context.Background(), "canary", metav1.GetOptions{})
 			o.Expect(err).NotTo(o.HaveOccurred(), "canary route not found")
+			host := ""
+			for _, ingress := range route.Status.Ingress {
+				if strings.Contains(ingress.Host, domain) {
```

Comment:
I'm not 100% sure what form it has, but it feels like `strings.HasSuffix()` is a better choice?

Reply:
Yeah, it has the following format for the host: `canary-openshift-ingress-canary.sharding-test.apps.shiftstack.com`

Reply:
done